concent_api/core/tasks.py

```diff
@@ -201,12 +205,15 @@ def verification_result(
     if subtask.next_deadline < parse_timestamp_to_utc_datetime(get_current_utc_timestamp()):
         task_to_compute = deserialize_message(subtask.task_to_compute.data.tobytes())
         # Worker makes a payment from requestor's deposit just like in the forced acceptance use case.
-        bankster.finalize_payment(
+        finalize_deposit_claim(
             subtask_id=subtask_id,
             concent_use_case=ConcentUseCase.ADDITIONAL_VERIFICATION,
```

Comment:
Again, this Celery task has `@atomic`. `finalize_deposit_claim()` won't be able to commit its transaction.

Reply:
:+1: done